source/manual/get-started.html.md

```diff
@@ -243,6 +243,26 @@ Test your SSH access by running:
 gds govuk connect --environment integration ssh backend
 ```
 
+If you see an error `Permission denied`, check the message shown later, similar to: `The SSH username used was: jsmith` - if this is not the user you specified in the puppet config above, you need to specify a username:
+
+```bash
+USER=jaysmith gds govuk connect --environment integration ssh backend
+```
+
+(or you can `export USER=jaysmith` separately to set it for a shell session)
```

Comment:
The above does duplicate some docs we have [here]https://github.com/alphagov/govuk-docker/blob/main/docs/troubleshooting.md#binsetup-fails-to-succeed) - not the end of the world, but it's nice to not create multiple places where we have to maintain docs! I wonder if we can consolidate these docs so that they're in the most appropriate location and easy to search for?